internal/confmapprovider/discovery/bundle/bundle.d/receivers/rabbitmq.discovery.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+{{ receiver "rabbitmq" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)rabbitmq.*"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)rabbitmq.*" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)rabbitmq.*"
+  config:
+    default:
+      endpoint: http://localhost:15672
```

Comment:
should this not be the discovered endpoint like  \`endpoint\` instead of hardcoding localhost?

Reply:
this is not at all needed. Not sure I have it here.